rhc-ose-ansible/inventory/ose-provision

```diff
@@ -12,6 +12,33 @@ openstack_key_name=""
 #openshift_openstack_master_storage_size: 10
 #openshift_openstack_node_storage_size: 10
 
+#env_id=
+rhc_ose_inv_dest=.
+rhc_ose_cluster_method=native
+rhc_ose_cluster_public_hostname=public.hostname.com
+rhc_ose_cluster_hostname=hostname.com
+#rhc_ose_ssh_user=root
+#rhc_ose_identity_provider=htpasswd
+deployment_type=openshift-enterprise
+
+#Openshift Components
+rhc_ose_enable_efk=true                  #If E-F-K logging stack should be deployed
```

Comment:
can we just use the standard vars from openshift-ansible for this? i.e. https://github.com/openshift/openshift-ansible/tree/master/playbooks/adhoc/metrics_setup